src/wallet/scriptpubkeyman.h

```diff
@@ -143,6 +145,17 @@ class CKeyPool
     }
 };
 
+class KeyIDHasher
+{
+public:
+    KeyIDHasher() {}
+
+    size_t operator()(const CKeyID& id) const
+    {
+        return id.GetUint64(0);
+    }
+};
+
```

Comment:
A few suggestions:

- drop the default constructor. The implicitly-defined default ctor will do exactly the same (ie nothing since this struct has no data members).
- make it a `struct` so you can drop the `public:` access specifier
- put the `operator()` definition on one line:

```suggestion
struct KeyIDHasher
{
    size_t operator()(const CKeyID& id) const { return id.GetUint64(0);}
};
```